lib/model/model.go

```diff
@@ -2204,6 +2426,12 @@ func (m *model) generateClusterConfig(device protocol.DeviceID) protocol.Cluster
 				Introducer:  deviceCfg.Introducer,
 			}
 
+			if deviceCfg.DeviceID == m.id && hasEncToken {
+				protocolDevice.EncPwToken = encToken
```

Comment:
It's a "thing" derived from the password, that allows everyone (whether they know the pw or not, i.e. trusted or not) to check that a folder is only every encrypted with one single pw and emit an error otherwise.

Reply:
Token implies it's used somehow, you exchange a token for something else.
This is more of a fingerprint.